orderbook/src/api/get_fee_and_price.rs

```diff
@@ -0,0 +1,186 @@
+use super::H160Wrapper;
+use crate::fee::{MinFeeCalculating, MinFeeCalculationError};
+use anyhow::Result;
+use chrono::{DateTime, Utc};
+use ethcontract::{H160, U256};
+use model::{order::OrderKind, u256_decimal};
+use num::BigInt;
+use serde::{Deserialize, Serialize};
+use shared::{
+    conversions::U256Ext,
+    price_estimate::{PriceEstimating, PriceEstimationError},
+};
+use std::convert::Infallible;
+use std::sync::Arc;
+use warp::{hyper::StatusCode, reply, Filter, Rejection, Reply};
+
+#[derive(Deserialize)]
+struct Query {
+    sell_token: H160Wrapper,
+    buy_token: H160Wrapper,
+    #[serde(with = "u256_decimal")]
+    amount: U256,
+    kind: OrderKind,
+}
+
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct Response {
+    expiration_date: DateTime<Utc>,
+    min_fee: U256,
+    #[serde(with = "serde_with::rust::display_fromstr")]
+    amount_after_fee: BigInt,
```

Comment:
This is the same as on the markets route, but isn't it a bit weird to have min_fee and amount_after_fee be different types?